sdk/src/utils.ts

```diff
@@ -302,3 +303,15 @@ const isArray = (value: any): boolean => {
 const isObject = (value: any): boolean => {
   return typeof value === 'object' && value !== null
 }
+
+export const constructTickmap = (tickmap: bigint[][]): bigint[] => {
+  const maxChunk = getMaxChunk()
+  const tmpTickmap = []
+  for (let i = 0; i <= maxChunk; i++) {
+    tmpTickmap[i] = 0n
+  }
+  for (let i = 0; i < tickmap.length; i++) {
+    tmpTickmap[Number(tickmap[i][0])] = tickmap[i][1]
+  }
+  return tmpTickmap
```

Comment:
Is this really `tmpTickmap`? It seems to be the result, so I suggest changing the naming, perhaps to `tickmap` and the parameter of the function as `initializedChunks`